docs/BUILD.md

```diff
@@ -1,65 +1,37 @@
-Build ipam-driver
-=================
-
-Prerequisite
-------------
-1. golang development environment is installed (https://golang.org/doc/install)
-
-
-Install Dependency
-------------------
-The driver primarily depends on libnetwork and the infoblox-go-client . They can be installed using the
-following commands:
+Building docker-ipam-plugin and docker-ipam-tools images
+========================================================
 
+Building Plugin Image
+---------------------
+To build the plugin image use the following command:
 ```
-go get github.com/docker/libnetwork  # libnetwork library (This also pulls down docker engine)
-go get github.com/docker/engine-api  # engine-api library (NOTE: run "make deps" to get dependencies)
-go get github.com/infobloxopen/infoblox-go-client  # Infoblox client
+make build-binary build-plugin create-plugin
```

Comment:
yes this can be , but I didn't do that for the ease of developer testing. I dont want to build a image every time to test changes to config.json. Or I dont need to build the binary again if no change is made.